src/openms/source/ANALYSIS/OPENSWATH/TransitionPQPFile.cpp

```diff
@@ -37,6 +37,36 @@
 namespace OpenMS
 {
 
+  bool columnExists(sqlite3 *db, const String& tablename, const String& colname)
```

Comment:
Would it make sense to move these kind of general methods to a SQLite helper class?

Reply:
yes, I will do that in a separate PR

Reply:
yes, I have done that now